packages/react-instantsearch-hooks-dom/src/widgets/__tests__/RangeInput.test.tsx

```diff
@@ -0,0 +1,179 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import {
+  createMultiSearchResponse,
+  createSearchClient,
+  createSingleSearchResponse,
+} from '../../../../../test/mock';
+import { InstantSearchHooksTestWrapper, wait } from '../../../../../test/utils';
+import { RangeInput } from '../RangeInput';
+
+const search = jest.fn((requests) => {
+  return Promise.resolve(
+    createMultiSearchResponse(
+      ...requests.map(() =>
+        createSingleSearchResponse({
+          facets: {
+            price: {},
+          },
+          facets_stats: {
+            price: {
+              min: 1,
+              max: 1000,
+              avg: 0,
+              sum: 0,
+            },
+          },
+        })
+      )
+    )
+  );
+});
+
+beforeEach(() => {
+  search.mockClear();
```

Comment:
How about we create a new function in each test so that we don't need to care about resetting the mock?